__tests__/components/editor/Browse.test.js

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 Stanford University see Apache2.txt for license
```

Comment:
I would change this date to 2019

Reply:
I updated browse and import resource template test files to 2019 copyright since these are ones that were just added.

Reply:
👍